p2p/proto/transaction.proto

```diff
@@ -15,10 +15,58 @@ message AccountSignature {
     repeated Felt252 parts = 1;
 }
 
+message L1HandlerV0 {
+    Felt252 nonce = 1;
+    Address address = 2;
+    Felt252 entry_point_selector = 3;
+    repeated Felt252 calldata = 4;
+}
+
+message DeclareV3Common {
+    Address sender = 1;
+    AccountSignature signature = 2;
+    Felt252 nonce = 4;
+    Hash compiled_class_hash = 5;
```

Comment:
`    Hash compiled_class_hash = 5;`

Is this a common field? As in, does RPC TX include this? I thought it was calculated by the Gateway from the Sierra code?

Reply:
We've made a decision to send data like hash objects even though it can be calculated from the rest of the fields. In our node implementation we will ignore this field
<!-- Reviewable comment -OBtcdQx40EHGX8L0hQ0:b34mg0t -->
<!-- Sent from Reviewable.io -->